src/shogun/classifier/mkl/MKL.cpp

```diff
@@ -241,14 +241,30 @@ class CMKL::Self
 };
 
 CMKL::CMKL(CSVM* s) : CSVM(), svm(NULL), C_mkl(0), mkl_norm(1), ent_lambda(0),
-		mkl_block_norm(1),beta_local(NULL), mkl_iterations(0), mkl_epsilon(1e-5),
+		mkl_block_norm(1),beta_local(NULL),beta_local_size(0), mkl_iterations(0), mkl_epsilon(1e-5),
 		interleaved_optimization(true), w_gap(1.0), rho(0), self()
 {
 	set_constraint_generator(s);
 	self->init();
 
 	SG_DEBUG("creating MKL object %p\n", this)
 	lp_initialized = false ;
+
+	// Register parameters
+	SG_ADD((CSGObject**) &svm, "svm", "wrapper svm", MS_NOT_AVAILABLE);
```

Comment:
this needs to be a in a `register_params` method that is called from every constructor